src/scripts/retropgf5-live-data/retropgf5-live-data.ts

```diff
@@ -99,10 +103,21 @@ export async function fetchAndProcessData() {
             return urls;
         }));
         const concatenatedUrls = [].concat(...urlArrays);
+
         try {
-            const responses = await Promise.all(concatenatedUrls.map(url => axios.get(url)));
-            const dataArray = responses.map(response => response.data);
-            return dataArray;
+            const responses: any[] = [];
+            await Promise.all(concatenatedUrls.map(async (url) => {
+                try {
+                    const response = await axios.get(url.metadataURL);
```

Comment:
Please add type for response for this `axios.get(url.metadataURL)` (maybe change schema or model in the future)